music21/roman.py

```diff
@@ -917,6 +917,84 @@ def romanNumeralFromChord(chordObj,
 
 
 class Minor67Default(enum.Enum):
+    '''
+    Showing how sixthMinor affects the interpretation of `vi`
```

Comment:
Probably needs to have a little introduction like, "Enumeration that can be passed into RomanNumeral creation to alter how roman numerals on the sixth and seventh scale degrees are parsed in minor."